WordPress/src/main/java/org/wordpress/android/ui/jetpack/scan/ScanFragment.kt

```diff
@@ -76,6 +90,31 @@ class ScanFragment : Fragment(R.layout.scan_fragment) {
         ((recycler_view.adapter) as ScanAdapter).update(content.items)
     }
 
+    private fun SnackbarMessageHolder.showSnackbar() {
+        val snackbar = WPSnackbar.make(
+            scan_state_layout,
+            uiHelpers.getTextOfUiString(requireContext(), this.message),
```

Comment:
Minor (🔍): The `this` keyword can be removed from the `message`.

Reply:
In 1fa403e